<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Теория расписаний</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Расписание</string>
    </property>
    <addaction name="actionNewSchedule"/>
    <addaction name="actionSaveSchedule"/>
    <addaction name="actionOpenSchedule"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="actionUndoChange"/>
    <addaction name="actionRedoChange"/>
    <addaction name="actionDetachAllTasks"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Алгоритм</string>
    </property>
    <addaction name="actionStartAlekseev"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionAboutThisProgram"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewSchedule">
   <property name="text">
    <string>Создать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSaveSchedule">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpenSchedule">
   <property name="text">
    <string>Загрузить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionUndoChange">
   <property name="text">
    <string>Отменить действие</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedoChange">
   <property name="text">
    <string>Повторить действие</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionStartAlekseev">
   <property name="text">
    <string>Запуск алгоритма Алексеева</string>
   </property>
  </action>
  <action name="actionDetachAllTasks">
   <property name="text">
    <string>Разгрузить всех исполнителей</string>
   </property>
  </action>
  <action name="actionAboutThisProgram">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
